src/open_inwoner/conf/locale/nl/LC_MESSAGES/django.po

```diff
@@ -17,159 +17,6 @@ msgstr ""
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 
-#: digid_eherkenning_oidc_generics/admin.py:13
-msgid "Activation"
-msgstr "Activatie"
-
-#: digid_eherkenning_oidc_generics/admin.py:17
-msgid "Common settings"
-msgstr "Algemene settings"
-
-#: digid_eherkenning_oidc_generics/admin.py:32
-msgid "Endpoints"
-msgstr "Endpoints"
-
-#: digid_eherkenning_oidc_generics/admin.py:44
-msgid "Keycloak specific settings"
-msgstr "Keycloak specifieke settings"
-
-#: digid_eherkenning_oidc_generics/admin.py:46
-msgid "Advanced settings"
-msgstr "Geavanceerde settings"
-
-#: digid_eherkenning_oidc_generics/apps.py:7
-msgid "DigiD & eHerkenning via OpenID Connect"
-msgstr "DigiD & eHerkenning via OpenID Connect"
-
-#: digid_eherkenning_oidc_generics/models.py:32
-msgid "Logout endpoint"
```

Comment:
@swrichards Deze hele zwik, die niet mee kwam in de nieuwe vertalingen, lijkt de boosdoener? Maar dan zouden we forever eerst `generate_config_docs` moeten draaien voordat we `makemessages -l nl ` starten?